listeners/src/main/java/org/fao/geonet/listener/metadata/draft/DraftUtilities.java

```diff
@@ -157,6 +156,9 @@ public AbstractMetadata replaceMetadataWithDraft(AbstractMetadata md, AbstractMe
 
             Log.info(Geonet.DATA_MANAGER, "Record updated with draft contents: " + md.getId());
```

Comment:
Can you update this log message so that it includes the source and target metadata id in the message.

"Record '" + md.getUuid() + "(" +md.getId() +")' update with draft contents from metadata id '" + draft.getId() +"'."

Reply:
fixed.